plugins/FreeBoy/FreeBoy.cpp

```diff
@@ -467,39 +467,30 @@ FreeBoyInstrumentView::FreeBoyInstrumentView( Instrument * _instrument,
 	m_ch1SweepTimeKnob->setHintText( tr( "Sweep Time:" ), "" );
 	m_ch1SweepTimeKnob->move( 5 + 4*32, 106 );
 	ToolTip::add( m_ch1SweepTimeKnob, tr( "Sweep Time" ) );
-	m_ch1SweepTimeKnob->setWhatsThis( tr( "The amount of increase or"
-									" decrease in frequency" ) );
 
 	m_ch1SweepRtShiftKnob = new FreeBoyKnob( this );
 	m_ch1SweepRtShiftKnob->setHintText( tr( "Sweep RtShift amount:" )
 										, "" );
 	m_ch1SweepRtShiftKnob->move( 5 + 3*32, 106 );
 	ToolTip::add( m_ch1SweepRtShiftKnob, tr( "Sweep RtShift amount" ) );
```

Comment:
These tooltips are very vague, I think the whatsthis text(or a shorter version of it) should become the tooltip text.

Reply:
Yes. Many tooltips will be updated, and created.